packages/gatsby-plugin-page-creator/src/derive-path.ts

```diff
@@ -51,6 +51,7 @@ export function derivePath(
     const replaceSlashesValue = (nodeValue + ``).replace(/\//g, `(REPLACED)`)
     const slugifiedWithoutSlashesValue = slugify(replaceSlashesValue, {
       lower: true,
+      remove: /\./,
```

Comment:
Would this not need the global flag? (`/\./g`)

Reply:
Good catch on the review, turns out I didn't need this. I tested, but I also changed libs so this code is gone